app/helpers/games_helper.rb

```diff
@@ -1,2 +1,17 @@
 module GamesHelper
-end
+	def color_block(row, col)
+		if ((row % 2 == 0) && (col % 2 != 0)) || ((row % 2 != 0) && (col % 2 == 0))
+			"wht"
+		else
+			"blk"
+		end
+	end
+
+	def assign_piece(pieces, row, col)
+		pieces.each do |piece|
+			if (piece.row_position) == row && (piece.col_position == col)
+				"#{piece.type}"
```

Comment:
Great start!  Find some images and see if you can figure out a way to display the image.  
HINT - 2 ways to do it are to assign a class to the <td> such as class='black-king', then use CSS to show the correct image.  The other way could be to add and image link here.  <img src="     " />  If you name your images consistantly it is easy to make this content dynamic. e.g. white-King.png   ( "#{piece.color}-#{piece.type}.png" )